crates/rolldown/tests/esbuild/dce/tree_shaking_import_identifier/artifacts.snap

```diff
@@ -22,13 +22,10 @@ class Base {
 //#region a.js
 var a_ns = {};
 __export(a_ns, {
-	Keep:() => Keep,
-	REMOVE:() => REMOVE
```

Comment:
Only keep is used.
here is the result of webpack: 
```js
/******/ (() => { // webpackBootstrap
/******/ 	"use strict";
var __webpack_exports__ = {};

;// CONCATENATED MODULE: ./a/b.mjs
class Base {}

;// CONCATENATED MODULE: ./a/a.mjs

class Keep extends Base {}
class REMOVE extends Base {}


;// CONCATENATED MODULE: ./a/index.js

new Keep();

/******/ })()
;

```

Reply:
the namespace object ref will be eliminated after we merge https://github.com/rolldown/rolldown/pull/1381